### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by svanellewee**
Hello there! Sorry to bother, but I noticed the sine wave example didn't match the module.. 

Thought I'd make a PR, this is a very cool project. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ as a sine wave:
 
 ```python
 import autopy
+import autopy.screen
```
Comment (msanders) [inline README.md]: ^ I think this bit should be extraneous

**Comment by svanellewee** [comment]
thanks for the feedback! Modified as you suggested.

I checked again and saw the example! Tried running it, then decided I'm going to add a python3 check for the example and make the docs match it instead.

Hope it's alright ?

**Comment by svanellewee** [comment]
Oh dear! I just realised that change doesnt make sense. Fixing now.

**Comment by svanellewee** [comment]
Squashed the commits, I hope the example change is in order.

File: README.md [hunk 2]
```diff
@@ -84,6 +84,10 @@ import autopy
 import math
 import time
 import random
+import sys
+
+if sys.version_info.major == 3:
+    xrange = range
```
Comment (msanders) [inline README.md]: I think using range as you had originally works well for this

File: examples/sine_mouse_wave.py [hunk 3]
```diff
@@ -2,20 +2,26 @@
...
+
+
 def sine_mouse_wave():
     """
     Moves mouse in a sine wave from the left edge of the screen to the right.
     """
     width, height = autopy.screen.size()
     height /= 2
-    height -= 10 # Stay in the screen bounds.
+    height -= 10  # Stay in the screen bounds.
```
Comment (msanders) [inline examples/sine_mouse_wave.py]: Thanks for fixing this

**Comment by msanders** [comment]
Thanks!

**Pull request merged**